### PR TITLE
CUMULUS-3848: Back ports CUMULUS-3841 and release updates for v18.3.4

### PR DESCRIPTION
**Pull request by paulpilone**
**Summary:** Summary of changes

Addresses [CUMULUS-3848: Back port and release CUMULUS-3841 to v18.3.4](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3848)

## Changes

* Backports CUMULUS-3841 to new version v18.3.4
* Adds v18.3.4 release updates

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: website/versions.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [
+  "v18.3.4",
+  "v18.3.3",
```
Comment (paulpilone) [inline website/versions.json:3]: This looks to have been missed in the v18.3.3 updates.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -45,3 +45,4 @@
 * Tim Clark
 * Tobias Nießen
 * Vanh Khuyen Nguyen
+* Vanh Nguyen
```
Comment (npauzenga) [inline CONTRIBUTORS.md:48]: Weird that npm generated another entry for Vanh. Not sure if this can be corrected for next time?

**Review by npauzenga — APPROVED** [review]
Looks great @paulpilone. There's a contributor listed twice and I'm curious if we can fix that but it's not critical or blocking.

File: CONTRIBUTORS.md (hunk 2 above)
Comment (paulpilone) [reply CONTRIBUTORS.md:48]: I'll leave it for now (CI is running) but I'd have to look at the script to see how that's pulled. Curious if maybe a commit was associated with a different email address (I've done that occasionally if Github decides to use my personal vs work email) and therefore a different name?

**Pull request merged**